desktop/desktopentry/desktopentry.go

```diff
@@ -156,6 +157,8 @@ func parse(filename string, r io.Reader) (*DesktopEntry, error) {
 				de.OnlyShowIn = splitList(value)
 			case "NotShownIn":
 				de.NotShownIn = splitList(value)
+			case "NoDisplay":
+				de.NoDisplay = value == "true"
```

Comment:
It's [not valid](https://specifications.freedesktop.org/desktop-entry-spec/latest/value-types.html) in the spec, but GDeskopAppInfo, that relies on GKeyFile also allows `1` and `0`, and likely the upper case versions